extensions/cornerstone/src/panels/PanelMeasurement.tsx

```diff
@@ -129,15 +117,31 @@ export default function PanelMeasurementTable({
         </MeasurementTable>
         {additionalFindings.length > 0 && (
           <MeasurementTable
+            key="additional"
             data={additionalFindings}
             title="Additional Findings"
-            onClick={jumpToImage}
-            onDelete={removeMeasurement}
-            onToggleVisibility={toggleVisibilityMeasurement}
-            onToggleLocked={toggleLockMeasurement}
-            onRename={renameMeasurement}
-            // onColor={changeColorMeasurement}
+            {...onArgs}
+          >
+            <MeasurementTable.Header>
+              <>
+                <div>Hello World</div>
+              </>
+            </MeasurementTable.Header>
+            <MeasurementTable.Body />
+          </MeasurementTable>
+        )}
+        {untrackedFindings.length > 0 && (
```

Comment:
I think this might upset some people because we're imposing an untracked view on them. I was hoping people could choose what to see based on the filters they set through the customization service. Here, we simply loop over those filters and create the view, regardless of the number.

<MeasurementTable.Header>

</MeasurementTable.Header>

Reply:
Actually, that is what I wanted to have, I just wasn't quite there yet.  Let me develop this a bit mroe and will pass it by you.